nifpga/session.py

```diff
@@ -187,6 +187,9 @@ def _irq_ordinals_to_bitmask(self, ordinals):
             bitmask |= (1 << ordinal)
         return bitmask
 
+    WaitOnIrqsReturnValues = namedtuple('WaitOnIrqsReturnValues',
```

Comment:
I found it interesting we could define a variable in a class scope sets it under self. I tested this locally and it seems to work, and Pep 8 doesn't complain, so I guess I am okay with it!